source/ns_cmdline.c

```diff
@@ -54,19 +54,32 @@
 //#define TRACE_DEEP
 //#define TRACE_PRINTF
 
-
 #ifdef TRACE_PRINTF
 #undef tr_debug
 #define tr_debug(...) printf( __VA_ARGS__);printf("\r\n")
 #endif
 
+// Disable EXCLUDE_TRACE_DEFINES macro to enable trace prints
+#define EXCLUDE_TRACE_DEFINES		1
+#ifdef EXCLUDE_TRACE_DEFINES
+#undef mbed_trace_exclude_filters_set
+#define mbed_trace_exclude_filters_set(...)
```

Comment:
These macros allow suppressing the traces when they aren't needed, so you might as well just remove the mbed_trace_exclude_filters_set line in the init function. After that this macro doesn't need to be defined at all.

Reply:
I don't want to update the source code in a way that external mbed_trace library cannot be used again. Old mbed_trace library can be used with MBED_CLIENT_CLI_TRACE_ENABLE as defined.